### DISCUSSION TITLE
Running cadence web through NGINX

### DISCUSSION THREAD

**Discussion by jeffreykky** [uber/cadence#4667] [Q&A]
I'm trying to access to cadence web from remote host, but the web default fetch from http://localhost:8088/api instead of http://server:8088/api, causing the failure to display workflows on the web.

And is there a way to add a URL prefix to all routes? e.g. API / CSS / JS

**Comment by longquanzheng** [comment]
Seems like you have open an issue at https://github.com/uber/cadence-web/issues/440 
that issue is the best place for the discussion.
